api.example.yml

```
@@ -95,6 +95,10 @@ api:
       max_name_size: 100
       # Max amount of characters allowed in team description
       max_description_size: 1000
+    # Limits applied to contact
+    contact:
+      # Amount of time to wait for sending test notification via contact
```

Может так? Я бы черканул ещё что-то про то, зачем оно ждёт. Там же типа чтобы были какие-то предварительная статистика по контакту сразу?
```suggestion
      # Amount of time to wait before sending test notification via contact
```

---

Нене
Сначала отправляется уведомление а потом ждём 10 секунд, пока score контакта не поменяется

---

Получается, мы не ждём 10 секунд перед отправкой, а сначала отправляем нотификацию, а потом ждём

---

А напишешь тут об этом? А то не оч понятно по описанию

---

Описал процесс в комменте рядом